client/apps/edit/components/display/index.coffee

```diff
@@ -0,0 +1,139 @@
+      done: (src) =>
+        @article.save thumbnail_image: src
+        $($magImage).find('.image-upload-hidden-input').trigger 'change'
+
+    $socialImage = $('.edit-display--social .edit-display__image-upload')
+    new ImageUploadForm
+      el: $socialImage
+      src: @article.get('social_image')
+      remove: =>
+        @article.save social_image: null
+        $($socialImage).find('.image-upload-hidden-input').trigger 'change'
+      done: (src) =>
+        @article.save social_image: src
+        $($socialImage).find('.image-upload-hidden-input').trigger 'change'
+
+    $emailImage = $('.edit-display--email .edit-display__image-upload')
+    new ImageUploadForm
+      el: $emailImage
+      src: @article.get('email_metadata')?.image_url
+      remove: =>
+        emailMetadata = @article.get('email_metadata') or {}
+        emailMetadata.image_url = ''
+        @article.save email_metadata: emailMetadata
+        $($emailImage).find('.image-upload-hidden-input').trigger 'change'
+      done: (src) =>
+        emailMetadata = @article.get('email_metadata') or {}
+        emailMetadata.image_url = src
+        @article.save email_metadata: emailMetadata
+        $($emailImage).find('.image-upload-hidden-input').trigger 'change'
+
+    $partnerImage = $('.edit-display--partner .edit-display__image-upload')
+    new ImageUploadForm
+      el: $partnerImage
+      src: @article.get('thumbnail_image')
+      remove: =>
+        @article.save thumbnail_image: null
+      done: (src) =>
+        @article.save thumbnail_image: src
+
+  prefillThumbnailTitle: =>
+    if @article.get('title') and not @article.get('thumbnail_title')
+      @useArticleTitle()
+
+  events:
+    'click .edit-display__use-article-title': 'useArticleTitle'
+    'change .edit-title-textarea': 'checkTitleInput'
+    'keyup .edit-display textarea': 'onKeyup'
+    'change .edit-display .image-upload-hidden-input': 'renderPreviews'
+    'click .edit-display__section-title': 'revealSection'
+
+  onKeyup: (e, initial = false) ->
+    @updateCharCount e
+    @renderPreviews e unless initial
+
+  updateCharCount: (e) ->
+    if e.target
+      e = e.target
+    textLength = $(e).data('limit') - e.value.length
+    if textLength < 0
+      $(e).parent().find('.edit-char-count').addClass('edit-char-count-limit')
+    else
+      $(e).parent().find('.edit-char-count').removeClass('edit-char-count-limit')
+    $(e).parent().find('.edit-char-count').text(textLength + ' Characters')
+
+  renderPreviews: (e, img) ->
+    # Pluck out the value that needs to be set
+    dataValue = $(e.target).data('value')
+    if dataValue in ['headline','author']
+      emailMetadata = @article.get('email_metadata') or {}
+      emailMetadata[dataValue] = $(e.target).val()
+      @article.set { email_metadata: emailMetadata }, silent: true
+    else if dataValue
+      @article.set { "#{dataValue}": $(e.target).val() }, silent: true
```

Comment:
Had no idea this existed! `silent:true` lets you make a change on the model and _not_ trigger a `change` event. This came up as an issue because since we are using the Article model to re-render previews, it was incorrectly telling Writer that the model changed even though we don't care. [This](https://github.com/artsy/positron/blob/master/client/apps/edit/components/header/index.coffee#L12) is where we were looking for those changes.